### PR TITLE
Feature/post new booking

### PR DESCRIPTION
**Pull request by kavakai**
What (if any) features are you implementing?
Add filter methods.
Add POST method (although I haven't tested it yet)

What (if anything) did you refactor?
I refactored quite a bit. Mostly CSS and some minor tweaking of my domUpdates methods.

Were there any issues that arose?
Took a while to get the filter to work, but it works.

Where should the reviewer start?
I would start with my scripts file at my event listeners, but mostly check the domUpdates methods.

Refactor checklist
[] Check variable and parameter names: are they descriptive and unique?
[] Is the code DRY? SRP?
[] Is it easy for others to read your code? (explanatory comments needed?)
[] Is the syntax consistent? (e.g. use of semicolons, blank lines)
[] Are commit messages formatted correctly?
goals?
Get my POST to work. 
Work on error messages and confirmation messages

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -32,6 +38,24 @@ const domUpdates = {
...
 
+    displayAvailableRooms(rooms) {
+        const allRoomsSection = document.getElementById('rooms');
+        allRoomsSection.innerHTML = '';
+        rooms.forEach(room => {
+            allRoomsSection.innerHTML += `
+            <article id="roomDisplay">
+                <h1 id="roomType">${room.roomType}</h1>
+                <li id="roomNum">${room.number}</li>
+                <li id="bidet">${room.bidet}</li>
```
Comment (Daniel-OC) [inline src/domUpdates.js:59]: Hell yeah! My only thought here is to consider finding ways of capitalizing things like roomType! (not needed per se, and definitely adjustable to style, but let me know if you need help doing it)

File: src/domUpdates.js [hunk 2]
```diff
@@ -74,14 +77,27 @@ const domUpdates = {
...
-    console.log("im confirming");
+    confirmBooking(event, rooms) {
+        const today = new Date().toISOString().split("T")[0];
+        const roomBook = rooms.find(room => event.target.id == room.number);
+        const booking = {
+            "userId": currentUser.id,
+            "date": today,
+            "roomNumber": roomBook.number
+        }
+        fetch("http://localhost:3001/api/v1/bookings", {
```
Comment (Daniel-OC) [inline src/domUpdates.js:94]: This looks right to me! Only follow up would be how are you updating the DOM with the new data?

File: src/domUpdates.js (hunk 1 above)
Comment (kavakai) [reply src/domUpdates.js:59]: That is super weird... my version has a completely different code here. That doesn't make sense why this pushed up, and not the version I have... Github can be maddening sometimes.

File: src/domUpdates.js (hunk 2 above)
Comment (kavakai) [reply src/domUpdates.js:94]: This is exactly where I left it last night, and I had horrible coding dreams about this exact issue. Todays project I suppose.

**Pull request merged**